README.md

```diff
@@ -4,39 +4,64 @@ This repository is used for distributing kotlin releases of the Matrix Rust SDK.
 
 ## Releasing
 
+There is a ['Release SDK Library (parallel)'](https://github.com/matrix-org/matrix-rust-components-kotlin/actions/workflows/release_sdk_parallel.yml) Github Action you can run with a parameter for the SDK branch or SHA - tags don't work correctly at the moment - and the new version name to release.
+
+If you want to do it manually instead:
+
 You need to set GITHUB_API_TOKEN in your env to let the script be able to commit and push, with this configuration:
 - Content: Read and Write;
 - Metadata: Read only.
 
-Make sures to also have setup the maven credentials and gpg key in your [env](scripts/publish-root.gradle) 
+Make sure to also have setup the maven credentials and gpg key in your [env](scripts/publish-root.gradle) 
 
 Whenever a new release of the underlying components is available, we need to tag a new release in this repo to make them available. 
 This is done with the release script found in the scripts directory. It'll also push the release to the maven repository.
 
 Usage : 
-
-`python3 ./scripts/release.py --version 0.1.3 --ref main --module SDK`
-
+```
+python3 ./scripts/release_sdk.py --version <version> --linkable-ref <sdk-branch/SHA> --module <SDK/CRYPTO>
```

Comment:
is it correct that this is used for releasing the crypto lib? (If so, what is `scripts/release_crypto.py` for?)